NEO4J_REFACTOR.md

```
@@ -0,0 +1,1088 @@
+
+4. **Event** - Time-based occurrences
+   - Properties: name, start_time, end_time, location, event_type
+   - Sources: Gmail meetings, GitHub releases, commit events, Google Calendar events
+
+5. **File** - Documents, code, media
+   - Properties: name, file_id, source, mime_type, size_bytes, url
+   - Sources: GitHub files, Gmail attachments, YouTube transcripts
+
+### Docker Compose Reader (12)
+
+6. **ComposeFile** (root entity)
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)